src/main/java/com/metamx/metrics/SysMonitor.java

```diff
@@ -71,6 +81,13 @@ public SysMonitor()
     );
   }
 
+  private void addDimensions(ServiceMetricEvent.Builder builder)
```

Comment:
Can we make this a static method somewhere so we don't have to copy the code for each Monitor?